ncm-spma/src/main/pan/components/spma/yum/schema.pan

```diff
@@ -65,6 +65,10 @@ type component_spma_yum = {
     "quattor_os_release" ? string # string to write to quattor_os_file
     "plugins" ? spma_yum_plugins
     "main_options" ? spma_yum_main_options
+    @{ Run the SPMA after configuring it }
+    "run" ? legacy_binary_affirmation_string
```

Comment:
I don't believe the yum backend supports this attribute so it should not appear here?